src/foam/nanos/notification/email/DAOResourceLoader.java

```diff
@@ -53,7 +55,9 @@ public static EmailTemplate findTemplate(X x, String name, String groupId, Strin
       Y     *     *     *
     */
 
+    logger.info("name : " + name + ", groupId : " + groupId + ", locale : " + locale + ", spid : " + spid);
```

Comment:
You don't need to concatenate in log messages. slows things down as well.   Logging is lazy so the string will only be concatenated if actually outputting this log level. 
```suggestion
    logger.info("name", name, "groupId", groupId, "locale", locale, "spid", spid);
```